packages/web-react/src/components/Header/Header.tsx

```diff
@@ -1,46 +1,21 @@
-import classNames from 'classnames';
 import React from 'react';
-import { useToggle } from '../../hooks';
-import { SpiritHeaderProps } from '../../types';
-import { HeaderProvider } from './HeaderContext';
-import { useHeaderStyleProps } from './useHeaderStyleProps';
-import { useStyleProps } from '../../hooks/styleProps';
+import classNames from 'classnames';
+import { useStyleProps } from '../../hooks';
+import { HeaderModernProps } from '../../types';
+import { useHeaderModernStyleProps } from './useHeaderStyleProps';
+import { HEADER_COLOR_DEFAULT } from './constants';
 
-const defaultProps = {
-  isInverted: false,
-  isSimple: false,
-};
+const Header = (props: HeaderModernProps) => {
```

Comment:
where we inforwe should also add deprecation hook as in Modal. One to the old one, where we inform the user, that the Header will be changed in the next version and they should migrate to HeaderModern. And one to the new one, where we inform the user, that the HeaderModern will be renamed to the Header in the next version. It is gonna be the same as in the Modal.

Reply:
done